client/src/components/SiteList.js

```diff
@@ -1,51 +1,49 @@
 import React, { useEffect, useState } from "react";
 import SiteTile from "./SiteTile";
 
-const SiteList = props => {
-    const [siteList, setSiteList] = useState([])
+const SiteList = (props) => {
+    const [siteList, setSiteList] = useState([]);
 
     const getSites = async () => {
         try {
-            const response = await fetch("/api/v1/sites")
+            const response = await fetch("/api/v1/sites");
             if (!response.ok) {
-                const errorMessage = `${response.status} (${response.statusText})`
-                const error = new Error(errorMessage)
-                throw(error) 
+                const errorMessage = `${response.status} (${response.statusText})`;
+                const error = new Error(errorMessage);
+                throw error;
             }
-            const responseBody = await response.json()
-            setSiteList(responseBody.siteList)
+            const responseBody = await response.json();
+            setSiteList(responseBody.siteList);
         } catch (error) {
-            console.error(`Error in Fetch: ${error.message}`)
+            console.error(`Error in Fetch: ${error.message}`);
         }
-    }
+    };
 
     useEffect(() => {
-        getSites()
-    }, [])
+        getSites();
+    }, []);
 
-    const sitesToRender = siteList.map(site => {
-        return(
+    const sitesToRender = siteList.map((site) => {
+        return (
             <li key={site.id}>
                 <SiteTile
```

Comment:
for this feature, the sites from the index page should ideally be clickable links. when building in "feature driven development", imagine as if when the branch is merged into `main` that all users of the app now have access to that feature. If this were merged in now, how would users know that changing the URL manually will being them to the details page?